faq.md

```diff
@@ -0,0 +1,24 @@
+# Frequently Asked Questions
+
+## What is the cost of certification?
+
+For [members](https://www.cncf.io/about/members/) of the Cloud Native Computing Foundation, there is no charge for
+certification. There is also no charge for non-profit organizations. For commercial organizations that wish to
+certify but don't want to become a CNCF member, the fee is the same as [joining](https://www.cncf.io/about/join/) the CNCF.
+
+## What about community distributions?
+
+For the purposes of certification, we treat community distributions/installers as non-profit organizations, so
+there is no charge. However, we do require an individual to complete the certification
+[agreement](https://github.com/cncf/k8s-conformance/blob/master/participation-form/Certified_Kubernetes_Form.md) so
+that we have an official contact if your software falls out of compliance.
```

Comment:
> an official contact

Can we change this in the future to make it possible for multiple points of contacts for a community-maintained project?

Reply:
> But we're certainly happy to get multiple contacts

Ok, cool. Please spell that out in this FAQ then for clarity for other OSS tools like kops?
We briefly discussed this during yesterday's SIG Cluster Lifecycle meeting and I think this would be a good message to get across to that OSS project for example

Reply:
@luxas Added the multiple contacts comment here: https://github.com/cncf/k8s-conformance/commit/36913ada179b300208343cfaa5140478e762f3f3#diff-4a1318da06786ac52626178ca960109eR15

Reply:
Great, thanks!